scripts/cpu-setonix.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash --login
+
+# SLURM directives
+#
+# Here we specify to SLURM we want an OpenMP job with 32 threads
+# a wall-clock time limit of one hour.
+#
+# Replace [your-project] with the appropriate project name
+# following --account (e.g., --account=project123).
+
+#SBATCH --account=pawsey0807
+#SBATCH --partition=work
+#SBATCH --exclusive=psharda
```

Comment:
If someone else tries to use it, I don't think it will work unless they change this to their username. But it should also work without, i.e. just `--exclusive`.